woeip/apps/core/factories.py

```diff
@@ -0,0 +1,11 @@
+import factory
+
+from woeip.apps.core.models import User
+
+
+class UserFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = User
+
+    name = factory.Faker('name')
```

Comment:
`User` doesn't have a `name` field. See https://github.com/clintonb/cookiecutter-django/pull/29/files for a complete factory.